### ISSUE TITLE
iina 1.2.0 在bigSur 11.2.3 中的 crash问题

### ISSUE THREAD

**Issue by Micro66** [iina/iina#3379]
**System and IINA version:**

- macOS bigSur 11.2.3
- IINA 1.2.0 

**Expected behavior:**

**Actual behavior:**

<details>
<summary><b>Crash report:</b></summary>

```
20:03:06.451 [player0][d] Open URL: file:///Users/xxxxxx/Downloads/xxxxxxxx.mp4
20:03:06.451 [player0][d] Opening /Users/xxxxxx/Downloads/xxxxxxxxx.mp4 in main window
2021-04-06 20:03:06.523 IINA[17049:35859380] This NSLayoutConstraint is being configured with a constant that exceeds internal limits.  A smaller value will be substituted, but this problem should be fixed. Break on BOOL _NSLayoutConstraintNumberExceedsLimit(void) to debug.  This will be logged only once.  This may break in the future.
20:03:06.611 [iina][d] Created OpenGL pixel format with [__C._CGLPixelFormatAttribute(rawValue: 5), __C._CGLPixelFormatAttribute(rawValue: 96), __C._CGLPixelFormatAttribute(rawValue: 99), __C._CGLPixelFormatAttribute(rawValue: 12800), __C._CGLPixelFormatAttribute(rawValue: 73), __C._CGLPixelFormatAttribute(rawValue: 101), __C._CGLPixelFormatAttribute(rawValue: 0)]
20:03:06.624 [iina][d] Falling back to nominal display refresh rate: 60.00082845465592 from 0.0
20:03:06.624 [iina][w] mpv log: [cplayer] warn: Setting the display-fps property is deprecated; set the override-display-fps property instead.
2021-04-06 20:03:06.661 IINA[17049:35859463] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'NSWindow drag regions should only be invalidated on the Main Thread!'
*** First throw call stack:
(
	0   CoreFoundation                      0x00007fff206356af __exceptionPreprocess + 242
	1   libobjc.A.dylib                     0x00007fff2036d3c9 objc_exception_throw + 48
	2   CoreFoundation                      0x00007fff2065da9a -[NSException raise] + 9
	3   AppKit                              0x00007fff22e04460 -[NSWindow(NSWindow_Theme) _postWindowNeedsToResetDragMarginsUnlessPostingDisabled] + 321
	4   AppKit                              0x00007fff22def121 -[NSWindow _initContent:styleMask:backing:defer:contentView:] + 1375
	5   AppKit                              0x00007fff22fb1377 -[NSPanel _initContent:styleMask:backing:defer:contentView:] + 50
	6   AppKit                              0x00007fff22deebbb -[NSWindow initWithContentRect:styleMask:backing:defer:] + 42
	7   AppKit                              0x00007fff22fb132c -[NSPanel initWithContentRect:styleMask:backing:defer:] + 64
	8   AppKit                              0x00007fff22deccd4 -[NSWindowTemplate nibInstantiate] + 393
	9   AppKit                              0x00007fff22db8af5 -[NSIBObjectData instantiateObject:] + 238
	10  AppKit                              0x00007fff22db8254 -[NSIBObjectData nibInstantiateWithOwner:options:topLevelObjects:] + 484
	11  AppKit                              0x00007fff22dacca8 loadNib + 392
	12  AppKit                              0x00007fff22dac2b0 +[NSBundle(NSNibLoading) _loadNibFile:nameTable:options:withZone:ownerBundle:] + 693
	13  AppKit                              0x00007fff22dabf06 -[NSBundle(NSNibLoading) loadNibNamed:owner:topLevelObjects:] + 201
	14  AppKit                              0x00007fff2313f6fe -[NSAlert init] + 137
	15  IINA                                0x0000000105c3a363 $s4IINA7UtilityC9showAlert_7comment9arguments5style11sheetWindowySS_SSSgSays7CVarArg_pGSgSo12NSAlertStyleVSo8NSWindowCSgtFZTf4nnnnnd_n + 83
	16  IINA                                0x0000000105cee3fc $s4IINA14SleepPreventerC07preventB0yyFZTf4d_n + 204
	17  IINA                                0x0000000105d56f32 $s4IINA10PlayerCoreC19checkStatusForSleepyyFZTf4d_n + 866
	18  IINA                                0x0000000105d69849 $s4IINA13MPVControllerC11handleEvent33_AA8C985EFDB89116DD68AB87EEE66A73LLyySPySo9mpv_eventVGSgF + 473
	19  IINA                                0x0000000105d70d0a $s4IINA13MPVControllerC10readEvents33_AA8C985EFDB89116DD68AB87EEE66A73LLyyFyycfU_TA + 42
	20  IINA                                0x0000000105d649de $sIeg_IeyB_TR + 14
	21  libdispatch.dylib                   0x00007fff203175dd _dispatch_call_block_and_release + 12
	22  libdispatch.dylib                   0x00007fff203187c7 _dispatch_client_callout + 8
	23  libdispatch.dylib                   0x00007fff2031e5fe _dispatch_lane_serial_drain + 606
	24  libdispatch.dylib                   0x00007fff2031f0cb _dispatch_lane_invoke + 375
	25  libdispatch.dylib                   0x00007fff20328c5d _dispatch_workloop_worker_thread + 819
	26  libsystem_pthread.dylib             0x00007fff204c0499 _pthread_wqthread + 314
	27  libsystem_pthread.dylib             0x00007fff204bf467 start_wqthread + 15
)
libc++abi.dylib: terminating with uncaught exception of type NSException
[1]    17049 abort      /Applications/IINA.app/Contents/MacOS/IINA
```
</details>

<br>

<details>
<summary><b>mpv log:</b></summary>

```
[   0.013][v][cplayer] mpv 0.32.0 Copyright © 2000-2020 mpv/MPlayer/mplayer2 projects
[   0.013][v][cplayer]  built on Sun Nov  8 00:37:38 CST 2020
[   0.013][v][cplayer] ffmpeg library versions:
[   0.013][v][cplayer]    libavutil       56.51.100
[   0.013][v][cplayer]    libavcodec      58.91.100
[   0.013][v][cplayer]    libavformat     58.45.100
[   0.013][v][cplayer]    libswscale      5.7.100
[   0.013][v][cplayer]    libavfilter     7.85.100
[   0.013][v][cplayer]    libswresample   3.7.100
[   0.013][v][cplayer] ffmpeg version: 4.3.1
[   0.013][v][cplayer] 
[   0.013][v][cplayer] Configuration: waf configure --prefix=/usr/local/Cellar/mpv-iina/0.32.0 --enable-javascript --enable-libmpv-shared --enable-lua --enable-libarchive --enable-uchardet --enable-libbluray --disable-swift --disable-debug-build --disable-macos-media-player --confdir=/usr/local/etc/mpv --datadir=/usr/local/Cellar/mpv-iina/0.32.0/share/mpv-iina --mandir=/usr/local/Cellar/mpv-iina/0.32.0/share/man --docdir=/usr/local/Cellar/mpv-iina/0.32.0/share/doc/mpv-iina --lua=51deb
[   0.013][v][cplayer] List of enabled features: 51deb aligned_alloc asm atomics bsd-fstatfs build-date cocoa coreaudio cplayer cplugins drmprime fchmod ffmpeg gl gl-cocoa glob glob-posix gnuc gpl iconv javascript jpeg lcms2 libarchive libass libass-osd libav-any libavcodec libavdevice libavutil libbluray libdl libm libmpv-shared lua macos-10-11-features macos-10-12-2-features macos-10-14-features macos-touchbar optimize osx-thread-name plain-gl posix posix-or-mingw posix-spawn posix-spawn-native pthreads rubberband stdatomic uchardet videotoolbox-gl videotoolbox-hwaccel zlib
[   0.013][v][cplayer] mpv 0.32.0 Copyright © 2000-2020 mpv/MPlayer/mplayer2 projects
[   0.013][v][cplayer]  built on Sun Nov  8 00:37:38 CST 2020
[   0.013][v][cplayer] ffmpeg library versions:
[   0.013][v][cplayer]    libavutil       56.51.100
[   0.013][v][cplayer]    libavcodec      58.91.100
[   0.013][v][cplayer]    libavformat     58.45.100
[   0.013][v][cplayer]    libswscale      5.7.100
[   0.013][v][cplayer]    libavfilter     7.85.100
[   0.013][v][cplayer]    libswresample   3.7.100
[   0.013][v][cplayer] ffmpeg version: 4.3.1
[   0.013][v][cplayer] 
[   0.013][v][cplayer] Configuration: waf configure --prefix=/usr/local/Cellar/mpv-iina/0.32.0 --enable-javascript --enable-libmpv-shared --enable-lua --enable-libarchive --enable-uchardet --enable-libbluray --disable-swift --disable-debug-build --disable-macos-media-player --confdir=/usr/local/etc/mpv --datadir=/usr/local/Cellar/mpv-iina/0.32.0/share/mpv-iina --mandir=/usr/local/Cellar/mpv-iina/0.32.0/share/man --docdir=/usr/local/Cellar/mpv-iina/0.32.0/share/doc/mpv-iina --lua=51deb
[   0.013][v][cplayer] List of enabled features: 51deb aligned_alloc asm atomics bsd-fstatfs build-date cocoa coreaudio cplayer cplugins drmprime fchmod ffmpeg gl gl-cocoa glob glob-posix gnuc gpl iconv javascript jpeg lcms2 libarchive libass libass-osd libav-any libavcodec libavdevice libavutil libbluray libdl libm libmpv-shared lua macos-10-11-features macos-10-12-2-features macos-10-14-features macos-touchbar optimize osx-thread-name plain-gl posix posix-or-mingw posix-spawn posix-spawn-native pthreads rubberband stdatomic uchardet videotoolbox-gl videotoolbox-hwaccel zlib
[   0.014][v][bdmv/bluray] Opening /Applications/IINA.app/Contents/Resources/config/iina-default-input.conf
[   0.014][v][file] Opening /Applications/IINA.app/Contents/Resources/config/iina-default-input.conf
[   0.014][d][file] resize stream to 131072 bytes
[   0.014][d][file] Stream opened successfully.
[   0.014][v][input] Parsing input config file /Applications/IINA.app/Contents/Resources/config/iina-default-input.conf
[   0.014][v][input] Input config file /Applications/IINA.app/Contents/Resources/config/iina-default-input.conf parsed: 69 binds
[   0.014][d][global] user path: '/Users/hongyu9/Library/Logs/com.colliderli.iina/2021-04-06-20-03-04_wq13xO/mpv.log' -> '/Users/hongyu9/Library/Logs/com.colliderli.iina/2021-04-06-20-03-04_wq13xO/mpv.log'
[   0.014][d][ytdl_hook] Loading lua script @ytdl_hook.lua...
[   0.015][d][stats] Loading lua script @stats.lua...
[   0.015][d][console] Loading lua script @console.lua...
[   0.015][d][osx] Exiting...
[   0.015][v][cplayer] Waiting for scripts...
[   0.015][d][stats] loading mp.defaults
[   0.015][d][console] loading mp.defaults
[   0.015][d][ytdl_hook] loading mp.defaults
[   0.015][v][cplayer] Set property: vo="libmpv" -> 1
[   0.015][v][cplayer] Set property: keepaspect="no" -> 1
[   0.016][v][cplayer] Set property: gpu-hwdec-interop="auto" -> 1
[   0.016][d][stats] loading @stats.lua
[   0.016][d][console] loading @console.lua
[   0.016][d][ytdl_hook] loading @ytdl_hook.lua
[   0.018][d][ytdl_hook] reading options for ytdl_hook 
[   0.018][d][ytdl_hook] script-opts/ytdl_hook.conf not found. 
[   0.018][d][ytdl_hook] lua-settings/ytdl_hook.conf not found. 
[   0.018][d][stats] reading options for stats 
[   0.018][d][stats] script-opts/stats.conf not found. 
[   0.018][d][stats] lua-settings/stats.conf not found. 
[   0.068][v][ao/coreaudio] selected audio output device:  (77)
[   0.072][d][console] reading options for console 
[   0.072][d][console] script-opts/console.conf not found. 
[   0.072][d][console] lua-settings/console.conf not found. 
[   0.073][d][cplayer] Run command: define-section, flags=64, args=["input_stats", "i script-binding stats/display-stats\nI script-binding stats/display-stats-toggle\n", "default"]
[   0.073][d][cplayer] Run command: enable-section, flags=64, args=["input_stats", "allow-hide-cursor+allow-vo-dragging"]
[   0.073][d][cplayer] Run command: define-section, flags=64, args=["input_forced_stats", "", "force"]
[   0.073][d][cplayer] Run command: enable-section, flags=64, args=["input_forced_stats", "allow-hide-cursor+allow-vo-dragging"]
[   0.081][v][cplayer] Done loading scripts.
[   0.082][v][osd/libass] Shaper: FriBidi 1.0.10 (SIMPLE) HarfBuzz-ng 2.7.2 (COMPLEX)
[   0.082][v][osd/libass] Setting up fonts...
[   0.085][v][osd/libass] Using font provider coretext
[   0.085][v][osd/libass] Done.
[   0.085][d][cplayer] Run command: define-section, flags=64, args=["input_console", "", "default"]
[   0.085][d][cplayer] Run command: enable-section, flags=64, args=["input_console", "allow-hide-cursor+allow-vo-dragging"]
[   0.085][d][cplayer] Run command: define-section, flags=64, args=["input_forced_console", "", "force"]
[   0.086][d][cplayer] Run command: enable-section, flags=64, args=["input_forced_console", "allow-hide-cursor+allow-vo-dragging"]
[   2.413][v][libmpv_render] GL_VERSION='4.1 INTEL-16.1.12'
[   2.413][v][libmpv_render] Detected desktop OpenGL 4.1.
[   2.413][v][libmpv_render] GL_VENDOR='Intel Inc.'
[   2.413][v][libmpv_render] GL_RENDERER='Intel(R) Iris(TM) Plus Graphics 645'
[   2.413][v][libmpv_render] GL_SHADING_LANGUAGE_VERSION='4.10'
[   2.413][d][libmpv_render] Combined OpenGL extensions string:
[   2.413][d][libmpv_render]  GL_ARB_blend_func_extended GL_ARB_draw_buffers_blend GL_ARB_draw_indirect GL_ARB_ES2_compatibility GL_ARB_explicit_attrib_location GL_ARB_gpu_shader_fp64 GL_ARB_gpu_shader5 GL_ARB_instanced_arrays GL_ARB_internalformat_query GL_ARB_occlusion_query2 GL_ARB_sample_shading GL_ARB_sampler_objects GL_ARB_separate_shader_objects GL_ARB_shader_bit_encoding GL_ARB_shader_subroutine GL_ARB_shading_language_include GL_ARB_tessellation_shader GL_ARB_texture_buffer_object_rgb32 GL_ARB_texture_cube_map_array GL_ARB_texture_gather GL_ARB_texture_query_lod GL_ARB_texture_rgb10_a2ui GL_ARB_texture_storage GL_ARB_texture_swizzle GL_ARB_timer_query GL_ARB_transform_feedback2 GL_ARB_transform_feedback3 GL_ARB_vertex_attrib_64bit GL_ARB_vertex_type_2_10_10_10_rev GL_ARB_viewport_array GL_EXT_debug_label GL_EXT_debug_marker GL_EXT_framebuffer_multisample_blit_scaled GL_EXT_texture_compression_s3tc GL_EXT_texture_filter_anisotropic GL_EXT_texture_sRGB_decode GL_APPLE_client_storage GL_APPLE_container_object_shareable GL_APPLE_flush_render GL_APPLE_object_purgeable GL_APPLE_rgb_422 GL_APPLE_row_bytes GL_APPLE_texture_range GL_ATI_texture_mirror_once GL_NV_texture_barrier
[   2.414][v][libmpv_render] Loaded extension GL_APPLE_rgb_422.
[   2.414][v][libmpv_render] GL_*_swap_control extension missing.
[   2.414][d][libmpv_render] Texture formats:
[   2.414][d][libmpv_render]   NAME       COMP*TYPE SIZE           DEPTH PER COMP.
[   2.414][d][libmpv_render]   r8         1*unorm   1B    LF CR ST {8}
[   2.414][d][libmpv_render]   rg8        2*unorm   2B    LF CR ST {8 8}
[   2.414][d][libmpv_render]   rgb8       3*unorm   3B    LF CR ST {8 8 8}
[   2.414][d][libmpv_render]   rgba8      4*unorm   4B    LF CR ST {8 8 8 8}
[   2.414][d][libmpv_render]   r16        1*unorm   2B    LF CR ST {16}
[   2.414][d][libmpv_render]   rg16       2*unorm   4B    LF CR ST {16 16}
[   2.414][d][libmpv_render]   rgb16      3*unorm   6B    LF CR ST {16 16 16}
[   2.414][d][libmpv_render]   rgba16     4*unorm   8B    LF CR ST {16 16 16 16}
[   2.414][d][libmpv_render]   r8ui       1*uint    1B       CR ST {8}
[   2.414][d][libmpv_render]   rg8ui      2*uint    2B       CR ST {8 8}
[   2.414][d][libmpv_render]   rgb8ui     3*uint    3B          ST {8 8 8}
[   2.414][d][libmpv_render]   rgba8ui    4*uint    4B       CR ST {8 8 8 8}
[   2.414][d][libmpv_render]   r16ui      1*uint    2B       CR ST {16}
[   2.414][d][libmpv_render]   rg16ui     2*uint    4B       CR ST {16 16}
[   2.414][d][libmpv_render]   rgb16ui    3*uint    6B          ST {16 16 16}
[   2.414][d][libmpv_render]   rgba16ui   4*uint    8B       CR ST {16 16 16 16}
[   2.414][d][libmpv_render]   r16f       1*float   4B    LF CR ST {32/16}
[   2.414][d][libmpv_render]   rg16f      2*float   8B    LF CR ST {32/16 32/16}
[   2.414][d][libmpv_render]   rgb16f     3*float  12B    LF CR ST {32/16 32/16 32/16}
[   2.414][d][libmpv_render]   rgba16f    4*float  16B    LF CR ST {32/16 32/16 32/16 32/16}
[   2.414][d][libmpv_render]   r32f       1*float   4B    LF CR ST {32}
[   2.414][d][libmpv_render]   rg32f      2*float   8B    LF CR ST {32 32}
[   2.414][d][libmpv_render]   rgb32f     3*float  12B    LF CR ST {32 32 32}
[   2.414][d][libmpv_render]   rgba32f    4*float  16B    LF CR ST {32 32 32 32}
[   2.414][d][libmpv_render]   rgb10_a2   4*unorm   4B    LF CR ST {0/10 0/10 0/10 0/2}
[   2.414][d][libmpv_render]   rgba12     4*unorm   8B    LF CR ST {16 16 16 16}
[   2.414][d][libmpv_render]   rgb10      3*unorm   6B    LF CR ST {16 16 16}
[   2.414][d][libmpv_render]   rgb565     3*unorm   2B    LF    ST {0/8 0/8 0/8}
[   2.414][d][libmpv_render]   appleyp    0*unorm   2B    LF    ST {}
[   2.414][d][libmpv_render]  LA = LUMINANCE_ALPHA hack format
[   2.414][d][libmpv_render]  LF = linear filterable
[   2.414][d][libmpv_render]  CR = can be used for render targets
[   2.414][d][libmpv_render]  ST = can be used for storable images
[   2.414][d][libmpv_render] Image formats:
[   2.414][d][libmpv_render]   yuv444p => 3 planes 1x1 8/0 [r8/r8/r8] (r/g/b)
[   2.414][d][libmpv_render]   yuv420p => 3 planes 2x2 8/0 [r8/r8/r8] (r/g/b)
[   2.414][d][libmpv_render]   gray => 1 planes 1x1 8/0 [r8] (r)
[   2.414][d][libmpv_render]   gray16 => 1 planes 1x1 16/0 [r16] (r)
[   2.414][d][libmpv_render]   uyvy422 => 1 planes 1x1 0/0 [appleyp] (brg)
[   2.414][d][libmpv_render]   nv12 => 2 planes 2x2 8/0 [r8/rg8] (r/gb)
[   2.414][d][libmpv_render]   p010 => 2 planes 2x2 16/6 [r16/rg16] (r/gb)
[   2.414][d][libmpv_render]   nv24 => 2 planes 1x1 8/0 [r8/rg8] (r/gb)
[   2.414][d][libmpv_render]   argb => 1 planes 1x1 8/0 [rgba8] (argb)
[   2.414][d][libmpv_render]   bgra => 1 planes 1x1 8/0 [rgba8] (bgra)
[   2.414][d][libmpv_render]   abgr => 1 planes 1x1 8/0 [rgba8] (abgr)
[   2.414][d][libmpv_render]   rgba => 1 planes 1x1 8/0 [rgba8] (rgba)
[   2.414][d][libmpv_render]   bgr24 => 1 planes 1x1 8/0 [rgb8] (bgr)
[   2.414][d][libmpv_render]   rgb24 => 1 planes 1x1 8/0 [rgb8] (rgb)
[   2.414][d][libmpv_render]   0rgb => 1 planes 1x1 8/0 [rgba8] (_rgb)
[   2.414][d][libmpv_render]   bgr0 => 1 planes 1x1 8/0 [rgba8] (bgr)
[   2.414][d][libmpv_render]   0bgr => 1 planes 1x1 8/0 [rgba8] (_bgr)
[   2.414][d][libmpv_render]   rgb0 => 1 planes 1x1 8/0 [rgba8] (rgb)
[   2.414][d][libmpv_render]   rgba64 => 1 planes 1x1 16/0 [rgba16] (rgba)
[   2.414][d][libmpv_render]   rgb565 => 1 planes 1x1 0/0 [rgb565] (rgb)
[   2.414][d][libmpv_render]   rgb30 => 1 planes 1x1 10/0 [rgb10_a2] (rgb)
[   2.414][d][libmpv_render]   vdpau
[   2.414][d][libmpv_render]   vdpau_output
[   2.414][d][libmpv_render]   vaapi
[   2.414][d][libmpv_render]   dxva2_vld
[   2.414][d][libmpv_render]   mmal
[   2.414][d][libmpv_render]   videotoolbox
[   2.414][d][libmpv_render]   drm_prime
[   2.414][d][libmpv_render]   yuyv422
[   2.414][d][libmpv_render]   yuv422p => 3 planes 2x1 8/0 [r8/r8/r8] (r/g/b)
[   2.414][d][libmpv_render]   yuv410p => 3 planes 4x4 8/0 [r8/r8/r8] (r/g/b)
[   2.414][d][libmpv_render]   yuv411p => 3 planes 4x1 8/0 [r8/r8/r8] (r/g/b)
[   2.414][d][libmpv_render]   monow
[   2.414][d][libmpv_render]   monob
[   2.414][d][libmpv_render]   pal8
[   2.414][d][libmpv_render]   yuvj422p => 3 planes 2x1 8/0 [r8/r8/r8] (r/g/b)
[   2.414][d][libmpv_render]   uyyvyy411
[   2.414][d][libmpv_render]   bgr8
[   2.414][d][libmpv_render]   bgr4
[   2.414][d][libmpv_render]   bgr4_byte
[   2.414][d][libmpv_render]   rgb8
[   2.414][d][libmpv_render]   rgb4
[   2.414][d][libmpv_render]   rgb4_byte
[   2.414][d][libmpv_render]   nv21 => 2 planes 2x2 8/0 [r8/rg8] (r/bg)
[   2.414][d][libmpv_render]   gray16be
[   2.414][d][libmpv_render]   yuv440p => 3 planes 1x2 8/0 [r8/r8/r8] (r/g/b)
[   2.414][d][libmpv_render]   yuvj440p => 3 planes 1x2 8/0 [r8/r8/r8] (r/g/b)
[   2.414][d][libmpv_render]   yuva420p => 4 planes 2x2 8/0 [r8/r8/r8/r8] (r/g/b/a)
[   2.414][d][libmpv_render]   rgb48be
[   2.414][d][libmpv_render]   rgb48 => 1 planes 1x1 16/0 [rgb16] (rgb)
[   2.414][d][libmpv_render]   rgb565be
[   2.414][d][libmpv_render]   rgb555be
[   2.414][d][libmpv_render]   rgb555
[   2.414][d][libmpv_render]   bgr565be
[   2.414][d][libmpv_render]   bgr565
[   2.414][d][libmpv_render]   bgr555be
[   2.414][d][libmpv_render]   bgr555
[   2.414][d][libmpv_render]   vaapi_moco
[   2.414][d][libmpv_render]   vaapi_idct
[   2.414][d][libmpv_render]   yuv420p16 => 3 planes 2x2 16/0 [r16/r16/r16] (r/g/b)
[   2.414][d][libmpv_render]   yuv420p16be
[   2.414][d][libmpv_render]   yuv422p16 => 3 planes 2x1 16/0 [r16/r16/r16] (r/g/b)
[   2.414][d][libmpv_render]   yuv422p16be
[   2.414][d][libmpv_render]   yuv444p16 => 3 planes 1x1 16/0 [r16/r16/r16] (r/g/b)
[   2.414][d][libmpv_render]   yuv444p16be
[   2.414][d][libmpv_render]   rgb444
[   2.414][d][libmpv_render]   rgb444be
[   2.414][d][libmpv_render]   bgr444
[   2.414][d][libmpv_render]   bgr444be
[   2.414][d][libmpv_render]   ya8 => 1 planes 1x1 8/0 [rg8] (ra)
[   2.414][d][libmpv_render]   bgr48be
[   2.414][d][libmpv_render]   bgr48 => 1 planes 1x1 16/0 [rgb16] (bgr)
[   2.414][d][libmpv_render]   yuv420p9be
[   2.414][d][libmpv_render]   yuv420p9 => 3 planes 2x2 16/-7 [r16/r16/r16] (r/g/b)
[   2.414][d][libmpv_render]   yuv420p10be
[   2.414][d][libmpv_render]   yuv420p10 => 3 planes 2x2 16/-6 [r16/r16/r16] (r/g/b)
[   2.414][d][libmpv_render]   yuv422p10be
[   2.414][d][libmpv_render]   yuv422p10 => 3 planes 2x1 16/-6 [r16/r16/r16] (r/g/b)
[   2.414][d][libmpv_render]   yuv444p9be
[   2.414][d][libmpv_render]   yuv444p9 => 3 planes 1x1 16/-7 [r16/r16/r16] (r/g/b)
[   2.414][d][libmpv_render]   yuv444p10be
[   2.414][d][libmpv_render]   yuv444p10 => 3 planes 1x1 16/-6 [r16/r16/r16] (r/g/b)
[   2.414][d][libmpv_render]   yuv422p9be
[   2.414][d][libmpv_render]   yuv422p9 => 3 planes 2x1 16/-7 [r16/r16/r16] (r/g/b)
[   2.414][d][libmpv_render]   gbrp => 3 planes 1x1 8/0 [r8/r8/r8] (g/b/r)
[   2.414][d][libmpv_render]   gbrp9be
[   2.414][d][libmpv_render]   gbrp9 => 3 planes 1x1 16/-7 [r16/r16/r16] (g/b/r)
[   2.414][d][libmpv_render]   gbrp10be
[   2.414][d][libmpv_render]   gbrp10 => 3 planes 1x1 16/-6 [r16/r16/r16] (g/b/r)
[   2.414][d][libmpv_render]   gbrp16be
[   2.414][d][libmpv_render]   gbrp16 => 3 planes 1x1 16/0 [r16/r16/r16] (g/b/r)
[   2.414][d][libmpv_render]   yuva422p => 4 planes 2x1 8/0 [r8/r8/r8/r8] (r/g/b/a)
[   2.414][d][libmpv_render]   yuva444p => 4 planes 1x1 8/0 [r8/r8/r8/r8] (r/g/b/a)
[   2.414][d][libmpv_render]   yuva420p9be
[   2.414][d][libmpv_render]   yuva420p9 => 4 planes 2x2 16/-7 [r16/r16/r16/r16] (r/g/b/a)
[   2.414][d][libmpv_render]   yuva422p9be
[   2.414][d][libmpv_render]   yuva422p9 => 4 planes 2x1 16/-7 [r16/r16/r16/r16] (r/g/b/a)
[   2.414][d][libmpv_render]   yuva444p9be
[   2.415][d][libmpv_render]   yuva444p9 => 4 planes 1x1 16/-7 [r16/r16/r16/r16] (r/g/b/a)
[   2.415][d][libmpv_render]   yuva420p10be
[   2.415][d][libmpv_render]   yuva420p10 => 4 planes 2x2 16/-6 [r16/r16/r16/r16] (r/g/b/a)
[   2.415][d][libmpv_render]   yuva422p10be
[   2.415][d][libmpv_render]   yuva422p10 => 4 planes 2x1 16/-6 [r16/r16/r16/r16] (r/g/b/a)
[   2.415][d][libmpv_render]   yuva444p10be
[   2.415][d][libmpv_render]   yuva444p10 => 4 planes 1x1 16/-6 [r16/r16/r16/r16] (r/g/b/a)
[   2.415][d][libmpv_render]   yuva420p16be
[   2.415][d][libmpv_render]   yuva420p16 => 4 planes 2x2 16/0 [r16/r16/r16/r16] (r/g/b/a)
[   2.415][d][libmpv_render]   yuva422p16be
[   2.415][d][libmpv_render]   yuva422p16 => 4 planes 2x1 16/0 [r16/r16/r16/r16] (r/g/b/a)
[   2.415][d][libmpv_render]   yuva444p16be
[   2.415][d][libmpv_render]   yuva444p16 => 4 planes 1x1 16/0 [r16/r16/r16/r16] (r/g/b/a)
[   2.415][d][libmpv_render]   xyz12 => 1 planes 1x1 16/4 [rgb16] (rgb)
[   2.415][d][libmpv_render]   xyz12be
[   2.415][d][libmpv_render]   nv16 => 2 planes 2x1 8/0 [r8/rg8] (r/gb)
[   2.415][d][libmpv_render]   nv20 => 2 planes 2x1 16/-6 [r16/rg16] (r/gb)
[   2.415][d][libmpv_render]   nv20be
[   2.415][d][libmpv_render]   rgba64be
[   2.415][d][libmpv_render]   bgra64be
[   2.415][d][libmpv_render]   bgra64 => 1 planes 1x1 16/0 [rgba16] (bgra)
[   2.415][d][libmpv_render]   yvyu422
[   2.415][d][libmpv_render]   ya16be
[   2.415][d][libmpv_render]   ya16 => 1 planes 1x1 16/0 [rg16] (ra)
[   2.415][d][libmpv_render]   gbrap => 4 planes 1x1 8/0 [r8/r8/r8/r8] (g/b/r/a)
[   2.415][d][libmpv_render]   gbrap16be
[   2.415][d][libmpv_render]   gbrap16 => 4 planes 1x1 16/0 [r16/r16/r16/r16] (g/b/r/a)
[   2.415][d][libmpv_render]   qsv
[   2.415][d][libmpv_render]   d3d11va_vld
[   2.415][d][libmpv_render]   cuda
[   2.415][d][libmpv_render]   yuv420p12be
[   2.415][d][libmpv_render]   yuv420p12 => 3 planes 2x2 16/-4 [r16/r16/r16] (r/g/b)
[   2.415][d][libmpv_render]   yuv420p14be
[   2.415][d][libmpv_render]   yuv420p14 => 3 planes 2x2 16/-2 [r16/r16/r16] (r/g/b)
[   2.415][d][libmpv_render]   yuv422p12be
[   2.415][d][libmpv_render]   yuv422p12 => 3 planes 2x1 16/-4 [r16/r16/r16] (r/g/b)
[   2.415][d][libmpv_render]   yuv422p14be
[   2.415][d][libmpv_render]   yuv422p14 => 3 planes 2x1 16/-2 [r16/r16/r16] (r/g/b)
[   2.415][d][libmpv_render]   yuv444p12be
[   2.415][d][libmpv_render]   yuv444p12 => 3 planes 1x1 16/-4 [r16/r16/r16] (r/g/b)
[   2.415][d][libmpv_render]   yuv444p14be
[   2.415][d][libmpv_render]   yuv444p14 => 3 planes 1x1 16/-2 [r16/r16/r16] (r/g/b)
[   2.415][d][libmpv_render]   gbrp12be
[   2.415][d][libmpv_render]   gbrp12 => 3 planes 1x1 16/-4 [r16/r16/r16] (g/b/r)
[   2.415][d][libmpv_render]   gbrp14be
[   2.415][d][libmpv_render]   gbrp14 => 3 planes 1x1 16/-2 [r16/r16/r16] (g/b/r)
[   2.415][d][libmpv_render]   yuvj411p => 3 planes 4x1 8/0 [r8/r8/r8] (r/g/b)
[   2.415][d][libmpv_render]   bayer_bggr8
[   2.415][d][libmpv_render]   bayer_rggb8
[   2.415][d][libmpv_render]   bayer_gbrg8
[   2.415][d][libmpv_render]   bayer_grbg8
[   2.415][d][libmpv_render]   bayer_bggr16
[   2.415][d][libmpv_render]   bayer_bggr16be
[   2.415][d][libmpv_render]   bayer_rggb16
[   2.415][d][libmpv_render]   bayer_rggb16be
[   2.415][d][libmpv_render]   bayer_gbrg16
[   2.415][d][libmpv_render]   bayer_gbrg16be
[   2.415][d][libmpv_render]   bayer_grbg16
[   2.415][d][libmpv_render]   bayer_grbg16be
[   2.415][d][libmpv_render]   xvmc
[   2.415][d][libmpv_render]   yuv440p10 => 3 planes 1x2 16/-6 [r16/r16/r16] (r/g/b)
[   2.415][d][libmpv_render]   yuv440p10be
[   2.415][d][libmpv_render]   yuv440p12 => 3 planes 1x2 16/-4 [r16/r16/r16] (r/g/b)
[   2.415][d][libmpv_render]   yuv440p12be
[   2.415][d][libmpv_render]   ayuv64 => 1 planes 1x1 16/0 [rgba16] (argb)
[   2.415][d][libmpv_render]   ayuv64be
[   2.415][d][libmpv_render]   p010be
[   2.415][d][libmpv_render]   gbrap12be
[   2.415][d][libmpv_render]   gbrap12 => 4 planes 1x1 16/-4 [r16/r16/r16/r16] (g/b/r/a)
[   2.415][d][libmpv_render]   gbrap10be
[   2.415][d][libmpv_render]   gbrap10 => 4 planes 1x1 16/-6 [r16/r16/r16/r16] (g/b/r/a)
[   2.415][d][libmpv_render]   mediacodec
[   2.415][d][libmpv_render]   gray12be
[   2.415][d][libmpv_render]   gray12 => 1 planes 1x1 16/-4 [r16] (r)
[   2.415][d][libmpv_render]   gray10be
[   2.415][d][libmpv_render]   gray10 => 1 planes 1x1 16/-6 [r16] (r)
[   2.415][d][libmpv_render]   p016 => 2 planes 2x2 16/0 [r16/rg16] (r/gb)
[   2.415][d][libmpv_render]   p016be
[   2.415][d][libmpv_render]   d3d11
[   2.415][d][libmpv_render]   gray9be
[   2.415][d][libmpv_render]   gray9 => 1 planes 1x1 16/-7 [r16] (r)
[   2.415][d][libmpv_render]   gbrpf32be
[   2.415][d][libmpv_render]   gbrpf32 => 3 planes 1x1 32/0 [r16f/r16f/r16f] (g/b/r)
[   2.415][d][libmpv_render]   gbrapf32be
[   2.415][d][libmpv_render]   gbrapf32 => 4 planes 1x1 32/0 [r16f/r16f/r16f/r16f] (g/b/r/a)
[   2.415][d][libmpv_render]   opencl
[   2.415][d][libmpv_render]   gray14be
[   2.415][d][libmpv_render]   gray14 => 1 planes 1x1 16/-2 [r16] (r)
[   2.415][d][libmpv_render]   grayf32be
[   2.415][d][libmpv_render]   grayf32 => 1 planes 1x1 32/0 [r16f] (r)
[   2.415][d][libmpv_render]   yuva422p12be
[   2.415][d][libmpv_render]   yuva422p12 => 4 planes 2x1 16/-4 [r16/r16/r16/r16] (r/g/b/a)
[   2.415][d][libmpv_render]   yuva444p12be
[   2.415][d][libmpv_render]   yuva444p12 => 4 planes 1x1 16/-4 [r16/r16/r16/r16] (r/g/b/a)
[   2.415][d][libmpv_render]   nv42 => 2 planes 1x1 8/0 [r8/rg8] (r/bg)
[   2.415][d][libmpv_render]   vulkan
[   2.415][d][libmpv_render]   y210be
[   2.415][d][libmpv_render]   y210
[   2.415][v][libmpv_render] Testing FBO format rgba16f
[   2.415][d][libmpv_render] Resizing texture: 16x16
[   2.415][v][libmpv_render] Using FBO format rgba16f.
[   2.415][v][libmpv_render] Disabling HDR peak computation (one or more of the following is not supported: compute shaders=0, SSBO=0).
[   2.415][v][libmpv_render] No advanced processing required. Enabling dumb mode.
[   2.415][v][libmpv_render] Loading hwdec driver 'videotoolbox'
[   2.417][w][cplayer] Setting the display-fps property is deprecated; set the override-display-fps property instead.
[   2.417][d][cplayer] Setting option 'override-display-fps' = 60.000828 (flags = 0) -> 0
[   2.417][v][cplayer] Set property: display-fps=60.000828 -> 1
[   2.420][v][cplayer] Set property: icc-profile="/Library/ColorSync/Profiles/Displays/Color LCD-64793C74-4F26-67F5-EB1F-8315E0C012E3.icc" -> 1
[   2.427][d][global] user path: '/Library/ColorSync/Profiles/Displays/Color LCD-64793C74-4F26-67F5-EB1F-8315E0C012E3.icc' -> '/Library/ColorSync/Profiles/Displays/Color LCD-64793C74-4F26-67F5-EB1F-8315E0C012E3.icc'
[   2.427][v][libmpv_render] Opening ICC profile '/Library/ColorSync/Profiles/Displays/Color LCD-64793C74-4F26-67F5-EB1F-8315E0C012E3.icc'
[   2.427][d][global] user path: '/Library/ColorSync/Profiles/Displays/Color LCD-64793C74-4F26-67F5-EB1F-8315E0C012E3.icc' -> '/Library/ColorSync/Profiles/Displays/Color LCD-64793C74-4F26-67F5-EB1F-8315E0C012E3.icc'
[   2.427][v][bdmv/bluray] Opening /Library/ColorSync/Profiles/Displays/Color LCD-64793C74-4F26-67F5-EB1F-8315E0C012E3.icc
[   2.427][v][file] Opening /Library/ColorSync/Profiles/Displays/Color LCD-64793C74-4F26-67F5-EB1F-8315E0C012E3.icc
[   2.427][d][file] resize stream to 131072 bytes
[   2.427][d][file] Stream opened successfully.
[   2.427][v][libmpv_render] Testing FBO format rgba16f
[   2.427][d][libmpv_render] Resizing texture: 16x16
[   2.427][v][libmpv_render] Using FBO format rgba16f.
[   2.427][v][libmpv_render] Disabling HDR peak computation (one or more of the following is not supported: compute shaders=0, SSBO=0).
[   2.427][v][libmpv_render] No advanced processing required. Enabling dumb mode.
[   2.438][d][cplayer] Run command: loadfile, flags=64, args=["/Users/hongyu9/Downloads/QQ20201202-153511.mp4", "replace", ""]
[   2.438][d][global] user path: '/Users/hongyu9/Library/Application Support/com.colliderli.iina/watch_later' -> '/Users/hongyu9/Library/Application Support/com.colliderli.iina/watch_later'
[   2.438][v][cplayer] Running hook: ytdl_hook/on_load
[   2.438][v][ytdl_hook] ytdl:// hook 
[   2.439][v][ytdl_hook] not a ytdl:// url 
[   2.439][v][bdmv/bluray] Opening /Users/hongyu9/Downloads/QQ20201202-153511.mp4
[   2.439][v][file] Opening /Users/hongyu9/Downloads/QQ20201202-153511.mp4
[   2.439][d][file] resize stream to 131072 bytes
[   2.439][d][file] Stream opened successfully.
[   2.439][v][demux] Trying demuxers for level=normal.
[   2.439][d][demux] Trying demuxer: disc (force-level: normal)
[   2.439][d][demux] Trying demuxer: edl (force-level: normal)
[   2.440][d][demux] Trying demuxer: cue (force-level: normal)
[   2.440][d][demux] Trying demuxer: rawaudio (force-level: normal)
[   2.440][d][demux] Trying demuxer: rawvideo (force-level: normal)
[   2.440][d][demux] Trying demuxer: mkv (force-level: normal)
[   2.440][d][demux] Trying demuxer: libarchive (force-level: normal)
[   2.440][d][demux] Trying demuxer: lavf (force-level: normal)
[   2.445][v][lavf] Found 'mov,mp4,m4a,3gp,3g2,mj2' at score=100 size=2048.
[   2.447][v][demux] Detected file format: mov,mp4,m4a,3gp,3g2,mj2 (libavformat)
[   2.447][v][cplayer] Opening done: /Users/hongyu9/Downloads/QQ20201202-153511.mp4
[   2.447][v][cplayer] Running hook: ytdl_hook/on_preloaded
[   2.447][v][lavf] select track 0
[   2.447][v][lavf] select track 1
[   2.447][i][cplayer]  (+) Video --vid=1 (*) (h264 480x408 15.036fps)
[   2.447][i][cplayer]  (+) Audio --aid=1 (*) (aac 2ch 44100Hz)
[   2.447][v][vo/libmpv] Assuming 60.000828 FPS for display sync.
[   2.448][v][vd] Container reported FPS: 15.036228
[   2.448][v][vd] Codec list:
[   2.448][v][vd]     h264 - H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10
[   2.448][v][vd] Opening decoder h264
[   2.448][v][vd] Looking at hwdec h264-videotoolbox...
[   2.448][v][vd] Trying hardware decoding via h264-videotoolbox.
[   2.448][v][vd] Selected codec: h264 (H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10)
[   2.448][v][vf] User filter list:
[   2.448][v][vf]   (empty)
[   2.449][v][ad] Codec list:
[   2.449][v][ad]     aac - AAC (Advanced Audio Coding)
[   2.449][v][ad]     aac_fixed (aac) - AAC (Advanced Audio Coding)
[   2.449][v][ad]     aac_at (aac) - aac (AudioToolbox)
[   2.449][v][ad] Opening decoder aac
[   2.449][v][ad] Detected 8 logical cores.
[   2.449][v][ad] Requesting 9 threads for decoding.
[   2.451][v][ad] Selected codec: aac (AAC (Advanced Audio Coding))
[   2.451][v][af] User filter list:
[   2.451][v][af]   (empty)
[   2.451][v][cplayer] Starting playback...
[   2.451][v][file] stream level seek from 266103 to 44
[   2.451][v][af] [in] 44100Hz stereo 2ch floatp
[   2.451][v][af] [userspeed] 44100Hz stereo 2ch floatp
[   2.451][v][af] [userspeed] (disabled)
[   2.451][v][af] [convert] 44100Hz stereo 2ch floatp
[   2.452][v][ao] Trying audio driver 'coreaudio'
[   2.452][v][ao/coreaudio] requested format: 44100 Hz, stereo channels, floatp
[   2.452][v][ao/coreaudio] selected audio output device:  (77)
[   2.452][v][ao/coreaudio] input channel layout:
[   2.452][v][ao/coreaudio] layout: tag: <0>, bitmap: <0>, descriptions <2>
[   2.452][v][ao/coreaudio]  - description 0: label <4294967295, 64>,  flags: <0>, coords: <0.000000, 0.000000, 0.000000>
[   2.452][v][ao/coreaudio]  - description 1: label <4294967295, 64>,  flags: <0>, coords: <0.000000, 0.000000, 0.000000>
[   2.452][v][ao/coreaudio] mp chmap: unknown2
[   2.453][v][ao/coreaudio] input channel layout:
[   2.453][v][ao/coreaudio] layout: tag: <0>, bitmap: <0>, descriptions <2>
[   2.453][v][ao/coreaudio]  - description 0: label <1, 0>,  flags: <0>, coords: <0.000000, 0.000000, 0.000000>
[   2.453][v][ao/coreaudio]  - description 1: label <2, 1>,  flags: <0>, coords: <0.000000, 0.000000, 0.000000>
[   2.453][v][ao/coreaudio] mp chmap: stereo
[   2.453][v][ao/coreaudio] Channel layouts:
[   2.453][v][ao/coreaudio]  - stereo
```
</details>

<br>

**Steps to reproduce:**

- [ ] MPV does not have this problem.

**How often does this happen?**
Open the video

**Comment by ksec** [comment]
https://github.com/iina/iina/issues/3364

**Comment by low-batt** [comment]
This:
```text
2021-04-06 20:03:06.661 IINA[17049:35859463] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'NSWindow drag regions should only be invalidated on the Main Thread!'
```

And this:
```text
15  IINA 0x0000000105c3a363 $s4IINA7UtilityC9showAlert_7comment9arguments5style11sheetWindowySS_SSSgSays7CVarArg_pGSgSo12NSAlertStyleVSo8NSWindowCSgtFZTf4nnnnnd_n + 83
16  IINA 0x0000000105cee3fc $s4IINA14SleepPreventerC07preventB0yyFZTf4d_n + 204
```

Positivity identifies this failure as the same one in issue #3478.
See that issue for full details on the problem.

**Comment by low-batt** [comment]
The fix in PR #3480 has been merged into the develop branch.

**Comment by low-batt** [comment]
Closing. Fixed in IINA 1.3.0.
